### PR TITLE
Fix computation of flush bounds

### PR DESCRIPTION
**Pull request by keithc-ca**
* don't assume pointers are 64-bits or platform is little-endian
* more carefully compute first and last cache lines to flush

### CODE REVIEW

**Comment by pshipton** [comment]
@mikezhang1234567890 fyi

File: runtime/jcl/common/sun_misc_Unsafe.cpp [hunk 1]
```diff
@@ -844,58 +844,65 @@ Java_jdk_internal_misc_Unsafe_objectFieldOffset1(JNIEnv *env, jobject receiver,
...
-				asm volatile("clflush %0" : "+m" (*endAddr));
-				break;
-			default:
-				goto error;
+		switch (vm->cpuCacheWritebackCapabilities) {
+		case J9PORT_X86_FEATURE_CLWB:
+			do {
+				cacheLine += cacheLineSize;
+				asm volatile("clwb %0" : "+m" (*(U_8 *)cacheLine));
+			}
```
Comment (gacholio) [inline runtime/jcl/common/sun_misc_Unsafe.cpp]: Please put the whiles on the brace line.

Comment (keithc-ca) [reply runtime/jcl/common/sun_misc_Unsafe.cpp]: Fixed.

**Comment by mikezhang1234567890** [comment]
lgtm

**Comment by gacholio** [comment]
As there are no tests for this feature at the moment, the CI compile is sufficient.

**Pull request merged**